docs/config.md

```diff
@@ -0,0 +1,128 @@
+# aeacus
+
+## Fields
+
+This is a list of configuration fields for `aeacus`.
+
+**name**: Image name, primarily used to organize remote scoring.
+
+> **Note!** This field is not mandatory if you use local scoring.
+
+```
+name = "ubuntu-18-dabbingdabbers"
+```
+
+**title**: Round title, as seen in the scoring report and README.
+
+```
+title = "CyberPatio Practice Round 1337"
+```
+
+**os**: Name of the operating system, as seen in the README.
+
+```
+os = "TempleOS 5.03"
+```
+
+**user**: Main user of the image. This is used when sending notifications.
+
+```
+user = "sysadmin"
+```
+
+**remote**: Address of remote server for scoring. If remote scoring is enabled, and `local` is not enabled, `aeacus` will refuse to score the image unless a connection to the server can be established.
+
+```
+remote = "http://scoring.example.com"
+```
+
+**password**: Password used for encrypting remote reporting traffic. The same password must be set on the remote side.
+
+```
+password = "H4!b5at+kWls-8yh4Guq"
+```
+
+**local**: Enables local scoring. If no remote address is specified, this will automatically be set to true.
+
+```
+local = true
+```
+
+**enddate**: Defines self-destruct date. If the engine is run after this date, the image will self destruct. Formatted as YEAR/MO/DA HR:MN:SC ZONE
+
+```
+enddate = "2004/06/05 13:09:00 PDT"
+```
+
+**destroy**: Governs self-destruct behavior. If this is set to true, the entire image will self-destruct, rather than just the `aeacus` folder.
```

Comment:
Can we add a note that this is currently deprecated?

Reply:
I wouldn't say it's deprecated, just not sufficiently tested that we're willing to bet nuking peoples' VMs. But I'd be down to remove it all together.